worker/migrations/migrate-accounts-to-organisation.go

```diff
@@ -185,12 +186,22 @@ func migrateAccountsToOrganization(domain string) error {
 	if err != nil {
 		return err
 	}
+	mu := lock.ReadWrite(inst, "migrate-accounts")
```

Comment:
`migrate-accounts` seems a bit too generic.
I would have named it like the migration : ` accounts-to-organization-migration` to avoid future collision.

Reply:
I have exact the opposite opinion: it is a good thing to also reuse the same lock for other migrations on the accounts to avoid CouchDB conflicts on those documents

Reply:
ok, that makes sense too.